src/utils/utils.js

```diff
@@ -45,3 +45,22 @@ exports.bubbleSort=(items)=>{
     return items;
 }
 
+exports.fibonacci=(firstElement,secondElement,n)=>{
```

Comment:
exported function should be in pascal case `getFibonacci`, 
**I am not getting the output** fix this